mailet/standard/src/test/java/org/apache/james/transport/matchers/HasMimeTypeTest.java

```diff
@@ -0,0 +1,60 @@
+package org.apache.james.transport.matchers;
+
+import org.apache.james.core.MailAddress;
+import org.apache.mailet.base.test.FakeMail;
+import org.apache.mailet.base.test.FakeMatcherConfig;
+import org.apache.mailet.base.test.MimeMessageBuilder;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.mail.internet.MimeMessage;
+
+import static org.junit.Assert.assertNotNull;
+
+public class HasMimeTypeTest {
+
+    private HasMimeType matcher;
+    private final String RECIPIENT = "test@james.apache.org";
+    private final String FROM = "test@james.apache.org";
+    private final String MIME_TYPES = "multipart/mixed";
```

Comment:
It's more MIME_TYPE

Reply:
You can specify multiple mime types in MIME_TYPES.